### ISSUE TITLE
Download link on Getting Started points to "todo"

### ISSUE THREAD

**Issue by dminkovsky** [thinkaurelius/titan#803]
On, http://s3.thinkaurelius.com/docs/titan/0.5.1/getting-started.html, the "Downloads" link points to "todo". I'd submit a PR, but docs in the repo currently point to the wiki, which appears to be for 0.4. Wasn't sure where to find 0.5 docs

**Comment by mbroecheler** [comment]
we'll fix it. Just for future reference, the docs are now in the docs/ directory of the project in the titan05 branch.

**Comment by dminkovsky** [comment]
@mbroecheler Thank you, yeah, it took me a while but I found that branch :). 

Related: there are links in the documentation to files on `master`; for example: in Chapter 3:

> Please see the GraphOfTheGodsFactory source code for details.

That links to the `master` version, which is illustrative but still pretty different from the relevant version on `titan05`.

**Comment by dalaro** [comment]
Thanks for pointing these things out.

I fixed the downloads link (25cf8f5b076fc74f817dd95b933cab8b7bbee89a).  I also grepped case-insensitively for todo in the docs markup, but I found no other particularly conspicuous hits like this one.

I found two references to Titan's blob/master and changed them to titan05 references (c12d1ebecbf423d4b9ae0b468263af4109218a45). A more elegant solution would probably involve an AsciiDoc property that's centrally controlled to avoid repetition, but with only two references, I don't want to go overboard.

I also cherry-picked these changes back to 0.5.0_docs and 0.5.1_docs branches that I've been keeping locally off their respective release tags and rebuilt+redeployed the docs for both of those versions.
